build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -180,12 +180,11 @@ steps:
   #   continueOnError: true
   #   condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
 
-  # {{SQL CARBON TODO}} - Reenable
-  # - script: |
-  #     set -e
-  #     yarn gulp vscode-linux-x64-build-deb
-  #   displayName: Build Deb
-  #   condition: and(succeeded(), ne(variables['EXTENSIONS_ONLY'], 'true'))
+  - script: |
```

Comment:
I would suggest working on finishing up Cheena's PR - which is just working to enable the build folder compile - first, and then afterwards can follow up with a separate PR to enable building the deb artifact. This will help make it easier to review and track down any issues that occur.

Reply:
Cheena's branch already compiles the build folder.

Reply:
Yeah, so my suggestion was to make sure that works fine and then merge it in so you can then just target this against main directly. @cheenamalhotra is there anything blocking that being completed?